packages/atlas-anatomogram-experiment-table/src/anatomogramExperimentTable.js

```diff
@@ -0,0 +1,305 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import Anatomogram from '@ebi-gene-expression-group/organ-anatomogram'
+import { TableManager } from '@ebi-gene-expression-group/atlas-experiment-table'
+import URI from 'urijs'
+
+const capitalizeFirstLetter = (str) => str.charAt(0).toUpperCase() + str.slice(1)
+
+
+
+class AnatomogramExperimentTable extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      selectedSpecies: props.species,
+      showIds: props.showIds,
+      experiments: props.experiments,
+      highlightIds: [],
+      selectIds: [],
+      selectAllIds: [],
+      selectedOrganId: ``,
+      resource: props.resource
+    }
+
+    this._addRemoveFromSelectIds = this._addRemoveFromSelectIds.bind(this)
+    this._showLinkBoxIds = this._showLinkBoxIds.bind(this)
+    this._clearSelectedIds = this._clearSelectedIds.bind(this)
+    this._handleSelectOnChange = this._handleSelectOnChange.bind(this)
+    this.handleResponse = this.handleResponse.bind(this)
+    this._afterSwitchView = this._afterSwitchView.bind(this)
+    this.getExperimentsAndUpdateState = this.getExperimentsAndUpdateState.bind(this)
+  }
+
+  _handleSelectOnChange(event) {
+    event.persist();
+    this.setState(() => ({
+      selectedSpecies: event.target.value
+    }))
+  }
+
+  _addRemoveFromSelectIds(ids) {
+    if (ids.length === 0 || ids[0].startsWith(`link`)) {
+      this.setState({
+        selectIds: ids
+      })
+    } else {
+      this.setState({
+        selectIds: ids,
+        resource:
+          URI(`json/experiments/hca/human`, this.props.host)
+            .search({ organismPart: ids })
+            .toString()
+      })
+    }
+    this.getExperimentsAndUpdateState()
+  }
+
+  async getExperimentsAndUpdateState() {
+    this.setState({
+      isLoading: true
+    })
+
+    const url = URI(this.props.resource, this.props.host).toString()

```

Comment:
This should be picking the resouce from the state, instead of props. Otherwise filtering by clickin on an organism part won’t be reflected in the table rows.

Reply:
Thank you for pointing this out, fixed that.